circle.yml

```diff
@@ -495,131 +495,99 @@ workflows:
   build-base-images:
     jobs:
       - build-base-image:
-          name: "base 10.22.0"
           dockerTag: "10.22.0"
       - build-base-image:
-          name: "base 12.0.0-libgbm"
           dockerTag: "12.0.0-libgbm"
           checkNodeVersion: false
       - build-base-image:
-          name: "base 12.18.3"
           dockerTag: "12.18.3"
       - build-base-image:
-          name: "base 12.18.4"
           dockerTag: "12.18.4"
       - build-base-image:
-          name: "base 12.19.0"
           dockerTag: "12.19.0"
       - build-base-image:
-          name: "base 14.10.1"
           dockerTag: "14.10.1"
       - build-base-image:
-          name: "base 14.15.0"
           dockerTag: "14.15.0"
       - build-base-image:
-          name: "base 14.15.4"
           dockerTag: "14.15.4"
       - build-base-image:
-          name: "base 14.16.0"
           dockerTag: "14.16.0"
       - build-base-image:
-          name: "base 14.17.0"
           dockerTag: "14.17.0"
       - build-base-image:
-          name: "base 14.17.3"
           dockerTag: "14.17.3"
       - build-base-image:
-          name: "base 14.18.1"
           dockerTag: "14.18.1"
       - build-base-image:
-          name: "base 14.5.0"
           dockerTag: "14.5.0"
       - build-base-image:
-          name: "base 14.7.0"
           dockerTag: "14.7.0"
       - build-base-image:
-          name: "base 16.0.0"
           dockerTag: "16.0.0"
       - build-base-image:
-          name: "base 16.1.0"
           dockerTag: "16.1.0"
       - build-base-image:
-          name: "base 16.13.0"
           dockerTag: "16.13.0"
       - build-base-image:
-          name: "base 16.2.0"
           dockerTag: "16.2.0"
       - build-base-image:
-          name: "base 16.3.0"
           dockerTag: "16.3.0"
       - build-base-image:
-          name: "base 16.4.0"
           dockerTag: "16.4.0"
       - build-base-image:
-          name: "base 16.5.0"
           dockerTag: "16.5.0"
       - build-base-image:
-          name: "base manjaro-14.12.0"
+          dockerTag: "17.3.0"
+      - build-base-image:
           dockerTag: "manjaro-14.12.0"
           checkNodeVersion: false
 
   build-browser-images:
     jobs:
       - build-browser-image:
-          name: "browsers node12.14.1-chrome85-ff81"
```

Comment:
name is not a parameter on the build-browser-image job. This value is not used.

Reply:
See my earlier comment.